app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -56,6 +67,28 @@ open class HomeActivity : AppCompatActivity() {
         })
     }
 
+    // Handles status bar theme change since the window does not dynamically recreate
+    private fun applyStatusBarTheme() {
+        window.statusBarColor = ContextCompat
+            .getColor(this, DefaultThemeManager
+                .resolveAttribute(android.R.attr.statusBarColor, this))
+
+        window.navigationBarColor = ContextCompat
+            .getColor(this, DefaultThemeManager
+                .resolveAttribute(android.R.attr.navigationBarColor, this))
+
+        when (themeManager.currentTheme) {
```

Comment:
This is really the only section I have doubts on (the rest has been reviewed/pair programmed). Flipping bits seems unclean, but the window does *not* get recreated when we make a new activity. Is there an easier way to just redraw this window? Perhaps @colintheshots knows 🤔

Reply:
@sblatz I think you can go back to the default value which is just 0 for both of those properties versus flipping the bits

Reply:
That was my first thought as well. Unfortunately doing that means they get *unset* when using a theme that sets them (as the window is not redrawn after)